docker/Dockerfile.ci

```
@@ -32,17 +32,27 @@ RUN apt-get update \
 	&& apt-get autoremove --yes \
 	&& rm -rf /var/lib/{apt,dpkg,cache,log}
 
-# Install bazel using the binary installer to enable building of flatc in the flatbuffers check
-RUN curl -Lo bazel-4.0.0-installer-linux-x86_64.sh https://github.com/bazelbuild/bazel/releases/download/4.0.0/bazel-4.0.0-installer-linux-x86_64.sh \
-  && chmod +x bazel-4.0.0-installer-linux-x86_64.sh \
-  && ./bazel-4.0.0-installer-linux-x86_64.sh \
-  && rm bazel-4.0.0-installer-linux-x86_64.sh
+# Install bazel using the installer script to enable building of flatc in the flatbuffers check
+ENV BAZEL_VERSION=4.0.0
+ENV BAZEL_DOWNLOAD_BASE="https://github.com/bazelbuild/bazel/releases/download"
+RUN curl https://bazel.build/bazel-release.pub.gpg | gpg --import - \
```

```suggestion
RUN curl --proto '=https' --tlsv1.2 -sSf https://bazel.build/bazel-release.pub.gpg | gpg --import - \
```

To ensure:

- use HTTPS for real
- enforce a sufficient TLS version
- silent mode
- show errors if they occur
- fail on HTTP errors

Taken from https://rustup.rs/ . Similar change should be applied to all `curl` invocations.